### PR TITLE
Implemented GetRecordingEdl

### PR DESCRIPTION
**Pull request by krustyreturns**

### CODE REVIEW

**Comment by krustyreturns** [comment]
background: https://forum.kodi.tv/showthread.php?tid=338806

File: pvr.wmc/addon.xml.in [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.wmc"
-  version="2.4.3"
+  version="2.4.5"
```
Comment (ksooo) [inline pvr.wmc/addon.xml.in]: Why is the version number increased by 2?Should read 2.4.4 imo.

File: pvr.wmc/changelog.txt [hunk 2]
```diff
@@ -1,4 +1,7 @@
-﻿2.4.0
+﻿2.4.5
```
Comment (ksooo) [inline pvr.wmc/changelog.txt]: 2.4.4

File: src/clientversion.h [hunk 3]
```diff
@@ -21,5 +21,5 @@
 
 inline std::string PVRWMC_GetClientVersion()
 {
-	return "2.4.0";	// ALSO CHANGE IN REV NUMBER in 'pvr.wmc/addon.xml.in'
+	return "2.4.5";	// ALSO CHANGE IN REV NUMBER in 'pvr.wmc/addon.xml.in'
```
Comment (ksooo) [inline src/clientversion.h]: 2.4.4

File: pvr.wmc/addon.xml.in (hunk 1 above)
Comment (sbthomas44) [reply pvr.wmc/addon.xml.in]: 2.4.4 was used as the version to test. I'm guessing to be safe it was incremented again to ensure any machines with the dev/test version of the add-on get the final version included with Kodi.

Comment (ksooo) [reply pvr.wmc/addon.xml.in]: Hmm, fairly uncommon as now  everybody is wondering what happend to  2.4.4.

Comment (krustyreturns) [reply pvr.wmc/addon.xml.in]: I'd like to keep the version to 2.4.5.  Basically the for the reason sbthomas said, but there are also comment  changes between the tested version and this one.

Comment (krustyreturns) [reply pvr.wmc/addon.xml.in]: Also...  I don't know how to 'squash down the changes to a single commit'.  If that's required I would need to do some research, or a little guidance.

File: pvr.wmc/changelog.txt (hunk 2 above)
Comment (krustyreturns) [reply pvr.wmc/changelog.txt]: I didn't think 2.4.4 needed to be there since it was the same as 2.4.5 and very few people used 2.4.4.

**Comment by ksooo** [comment]
???

**Comment by krustyreturns** [comment]
I gave up.  Will try to clean up with a new PR

**Pull request closed without merging**